model/docs/example_with_datasets.qmd

```diff
@@ -501,3 +484,58 @@ out = hosp_model_weekday.plot_posterior(
     ),
 )
 ```
+We can use ArviZ to visualize the posterior predictive distributions.
+```{python}
+# | label: posterior-predictive-distribution
+idata_weekday = az.from_numpyro(
+    hosp_model_weekday.mcmc,
+    posterior_predictive=hosp_model_weekday.posterior_predictive(n_timepoints_to_simulate=1)
+)
+```
+Below we plot the posterior predictive distributions using equal tailed bayesian credible intervals:
+
+```{python}
+# | label: fig-output-posterior-predictive
+# | fig-cap: Posterior Predictive Infections
+
+def compute_eti(dataset,eti_prob):
```

Comment:
Is there a better place for this function? It seems like it will be useful quite often. cc @dylanhmorris @gvegayon